caikit/runtime/service_generation/signature_parsing/parsers.py

```diff
@@ -79,7 +73,7 @@ def get_output_type_name(
     log.debug3("Parent module parts for %s: %s", module_class.__name__, module_parts)
 
     # TODO: this part needs a test (or consider deleting and say user needs to specify output type)
-    class_name = _snake_to_camel(module_parts[2]) + "Prediction"
+    class_name = _snake_to_camel(module_parts[2]) + "Predict"
```

Comment:
🤔 , so changing this here would actually affect any users relying on this functionality.

I think I'm leaning towards just cutting it entirely, this was one of those hacks to get up and running but users should be specifying return types at least in the docstring or as a type annotation.